### ISSUE TITLE
Unexpected Exception in vsim.gui.controllers.SimulatorController/lambda$null$24

### ISSUE THREAD

**Issue by rsrsoley** [andrescv/V-Sim#6]
**Stacktrace**:
```java
java.lang.NullPointerException
	at vsim.gui.controllers.SimulatorController.lambda$null$24(SimulatorController.java:572)
	at com.sun.javafx.application.PlatformImpl.lambda$null$172(PlatformImpl.java:295)
	at java.security.AccessController.doPrivileged(Native Method)
	at com.sun.javafx.application.PlatformImpl.lambda$runLater$173(PlatformImpl.java:294)
	at com.sun.glass.ui.InvokeLaterDispatcher$Future.run(InvokeLaterDispatcher.java:95)
	at com.sun.glass.ui.gtk.GtkApplication._runLoop(Native Method)
	at com.sun.glass.ui.gtk.GtkApplication.lambda$null$48(GtkApplication.java:139)
	at java.lang.Thread.run(Thread.java:748)
```

**Comment by andrescv** [comment]
Fixed in `v2.0.2`

**Comment by andrescv** [comment]
fixed in `v2.0.2`
